supplementary_files/handlers_stack/lambdas/invoked_by_apigw_cloudformation/lambda_function.py

```diff
@@ -214,20 +265,46 @@ def lambda_handler(event,context):
     
     # set response
     
+    evaluation_key = f'cb-{generate_uuid()}'
+    
     response_expected_by_consumer = {
-        "ResultsReport": {
-            "Key": f'cb-{generate_uuid()}',
-            "Buckets": {
-                "Raw": os.environ['RawPaCResultsBucket'],
-                "OutputHandlers":json.loads(os.environ['OutputHandlers'])
+        "ControlBrokerEvaluation": {
+            "Raw": {
+                "PresignedUrl": generate_presigned_url(
+                    bucket = os.environ['RawPaCResultsBucket'],
+                    key = evaluation_key
+                ),
+                "Bucket": os.environ['RawPaCResultsBucket'],
+                "Key": evaluation_key
+            },
+            "OutputHandlers":{
```

Comment:
For my own understanding, why does the input handler return an object of OutputHandlers to the end user? I guess to give the user all the available representations since we don't know which is actually suitable for them?